packages/contracts/src/dollar/interfaces/AggregatorV3Interface.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+// ChainLink price feed interface
+// @dev Original implementation https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol
+
+pragma solidity 0.8.19;
+
+interface AggregatorV3Interface {
```

Comment:
It makes sense to install https://github.com/smartcontractkit/chainlink as a dependency instead of copy-pasting the code.